src/services/settingsManager.ts

```diff
@@ -115,6 +103,28 @@ export class SettingsManager {
         description: "Toggle alias provider when completing module options",
       },
     },
+    validation: {
+      enabled: {
+        default: true,
+        description:
+          "Toggle validation provider. If enabled and ansible-lint is disabled, validation falls back to ansible-playbook --syntax-check",
```

Comment:
If validation is disabled, there should be no fall back on  `ansible-playbook --syntax-check`. If validation is enabled but `ansible-lint` is disabled in that case we can rely on `ansible-playbook --syntax-check` . 
As per my understanding, previous to this PR we did fallback because the setting to enable/disable validation was not available.

Reply:
Yes. I have written the same thing in the description.

Reply:
1. validation disabled -> no diagnostics
2. validation enabled + lint disabled -> diagnostics using ansible-playbook syntax check
3. validation enabled + lint enabled -> diagnostics using ansible lint